### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by dingzhouzhou-git** [langgenius/dify#7061] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I enabled the reference and attribution function in the workflow orchestration mode of the chat assistant, but I could not reference.
![Uploading 20240807182044.jpg…]()

### 2. Additional context or comments

![Uploading 20240807182044.jpg…]()

**Comment by crazywoola** [comment]
Images are.broken.
